extensions/wikia/Wall/WallMessage.class.php

```diff
@@ -691,18 +569,18 @@ public function getWallPageUrl() {
 	}
 
 	/**
-	 * @param bool $useMasterDB
 	 * @return Title
 	 */
 	public function getArticleTitle( $useMasterDB = false ) {
-		$commentsIndex = $this->getCommentsIndex();
-
+		$commentsIndex = $this->getCommentsIndexEntry();
 		if ( empty( $commentsIndex ) ) {
+			$this->error( 'SUS-1680 - No comments_index entry for message', [
```

Comment:
Let's prefix the message with `__METHOD__` to make it easier to find this place.